aspnetcore/fundamentals/openapi/openapi-comments.md

```diff
@@ -0,0 +1,300 @@
+
+This article includes a [sample app](#download10) that demonstrates the [`Microsoft.AspNetCore.OpenApi`](https://www.nuget.org/packages/Microsoft.AspNetCore.OpenApi) package's ability to integrate XML documentation comments on types into OpenAPI documents. The sample app is a minimal ASP.NET Core Web API project that uses the `Microsoft.AspNetCore.OpenApi` package to generate OpenAPI documents. The XML documentation comments are used to populate summaries, descriptions, parameter information, and response details in the generated OpenAPI document.
+
+![screenshot of app with XML comments in sclaar UI](~/fundamentals/openapi/_static/screenshot.png)
+
+## Customizing XML documentation behavior
+
+The following sections describe how to enable and customize XML documentation support.
+
+### Adding XML documentation sources
+
+The [`Microsoft.AspNetCore.OpenApi`](https://www.nuget.org/packages/Microsoft.AspNetCore.OpenApi) NuGet package automatically resolves XML documentation comments from:
+
+* The application assembly, when the `GenerateDocumentationFile` property is set. In the [sample app](#download10), the `API` project.
+
+  :::code language="xml" source="~/fundamentals/openapi/samples/10.x/aspnet-openapi-xml/models/Models.csproj" highlight="7":::
+
+* Any projects referenced via `ProjectReferences` that have the `GenerateDocumentationFile` property set. In the sample app, the `Models` project:
+
+  :::code language="xml" source="~/fundamentals/openapi/samples/10.x/aspnet-openapi-xml/api/Api.csproj" highlight="7,16":::
+
+The implementation discovers XML files statically at compile-time. The `AdditionalFiles` item group specifies additional sources for XML files:
+
+  :::code language="xml" source="~/fundamentals/openapi/samples/10.x/aspnet-openapi-xml/api/AdditionalFiles.xml" highlight="7":::
+
+#### Disabling XML documentation support
+
+To turn off XML documentation integration, remove the source generator from the `Analyzers` item group. Removing the source generator prevents it from being used during compilation.
+
+<!-- review: \aspnet-openapi-xml\api\obj\Debug\net10.0\Api.xml is generated using the following project file , that is, <Analyzer Remove="$(PkgMicro ... -->
+
+:::code language="xml" source="~/fundamentals/openapi/samples/10.x/aspnet-openapi-xml/api/Api-remove.csproj.xml" highlight="7,19-23":::
+
+<!-- review setting  <GenerateDocumentationFile>false</GenerateDocumentationFile> prevents the XML file from being generated.  Explain the difference between GenerateDocumentationFile = true and <Analyzer Remove -->
+
+## Source generator implementation notes
```

Comment:
I think we can remove the implementation notes section. I included them in my repo for those who are curious but we usually don't include things like this in the docs AFAIK.